packages/client/README.md

```diff
@@ -114,257 +205,280 @@ const msg = {
 ```js
-const sub = await client.subscribe(options)
-sub.on('resent', () => {
-    console.log('All caught up and received all requested historical messages! Now switching to real time!')
-})
+await stream.hasPermission({
+    permission: StreamPermission.PUBLISH,
+    user: '0x12345...',
+    allowPublic: true
+}
 ```
 
-### Storage
+All streams permissions can be queried by calling `stream.getPermissions()`:
+```js
+const permissions = await stream.getPermissions()
+```
+The returned permissions are an array containing an item for each user, and one for public permissions:
+```js
+    permissions = [
+        { user: '0x12345...', permissions: ['subscribe', 'publish'] },
+        { public: true, permissions: ['subscribe']}
+    ]
+```
 
-You can enable data storage on your streams to retain historical data in one or more geographic locations of your choice. By default storage is not enabled on streams.
 
-`await stream.addToStorageNode(StorageNode.STREAMR_GERMANY)`
+#### Deleting a stream
+Deletes the stream from the on-chain registry:
+```js
+// Requires gas
+await stream.delete()
+```
 
-## Stream API
 
-All the below functions return a Promise which gets resolved with the result.
 
-| Name                                                | Description                                                                                                                                          |
-| :-------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- |
-| getStream(streamId)                                 | Fetches a stream object from the API.                                                                                                                |
-| searchStreams(term)                                 | Fetches an array of stream objects      |
-| createStream(\[properties])                         | Creates a stream with the given properties. For more information on the stream properties, consult the [API docs](https://api-explorer.streamr.com). If you specify `id`, it can be a full streamId or a path (e.g. `/foo/bar` will create a stream with id `<your-ethereum-address>/foo/bar` if you have authenticated with a private key)|
-| getOrCreateStream(properties)                       | Gets a stream with the id given in `properties`, or creates it if one is not found.                                                          |
-| publish(streamId, message, timestamp, partitionKey) | Publishes a new message to the given stream.                                                                                                         |
+### Storage Options
 
-### Stream object
+You can enable data storage on your streams to retain historical data in one or more geographic locations of your choice and access it later via `resend`. By default storage is not enabled on streams. You can enable it with:
```

Comment:
There are no multiple geographic locations at the moment (just the storage node in Germany)